_install_guide/logging.md

```diff
@@ -133,3 +134,65 @@ docker inspect -f '{{.HostConfig.LogConfig}}' clouddriver
 {% endraw %}```
 
 If the log config isn't what you expect then something is wrong in the Armory Spinnaker config, and if the config is what you expect the problem is likely in your distributed logging setup.
+
+## Monitoring Spinnaker With Datadog
+
+Spinnaker provides a monitoring container which exports metrics from the core sub-services. Below we add two additional containers to our docker-compose setup: `datadog` and `spinnaker-monitoring`.
+
+Add the following to `/opt/spinnaker/compose/docker-compose.override.yml`
+```
+version: "2.1"
+services:
+  datadog:
+    container_name: datadog
+    hostname: datadog${HOSTNAME_SUFFIX}
+    env_file: /opt/spinnaker/config/datadog_api_token.txt
+    environment:
+      - "SD_BACKEND=docker"
+    image: datadog/docker-dd-agent:latest
+    ports:
+      - "8125:8125"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - /proc/:/host/proc/:ro
+      - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+  spinnaker-monitoring:
+    container_name: spinnaker-monitoring
+    hostname: spinnaker-monitoring${HOSTNAME_SUFFIX}
+    image: armory/spinnaker-monitoring:version-0.1.0
+    ports:
+      - "8008:8008"
+    volumes:
+      - /opt/spinnaker/config/monitoring/registry/:/opt/spinnaker-monitoring/registry/
+      - /opt/spinnaker/config/:/opt/spinnaker-monitoring/config/
+```
+
+In the configuration above `/opt/spinnaker/config/datadog_api_token.txt` is a properties file which contains your Datadog API key
+```
+API_KEY=${YOUR_DATADOG_API_KEY}
+```
+
+Add the Spinnaker monitoring configuration file in `/opt/spinnaker/config/spinnaker-monitoring.yml`:
```

Comment:
Can we just defaultly include this in armoryspinnaker package?

Reply:
We can, but I don't want to make this change right now as this is likely to change as we move to Kubernetes.